.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 composer.lock
 vendor
+/.idea
```

Comment:
You should put this line in your global gitignore, not in individual repositories (https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)